projects/1.20/assets/argentinasdelight/argentinas_delight/lang/zh_cn.json

```
@@ -0,0 +1,566 @@
+{
+  "block.argentinas_delight.aceitehirviendo": "一锅热油",
+  "block.argentinas_delight.achura_2click": "一盘烤牛肠(3/5)",
+  "block.argentinas_delight.achura_3click": "一盘烤牛肠(2/5)",
+  "block.argentinas_delight.achura_4click": "一盘烤牛肠(1/5)",
+  "block.argentinas_delight.achurahechabloque": "一份烤熟的牛肠",
+  "block.argentinas_delight.achuras_1click": "一盘烤牛肠(4/5)",
```

```suggestion
  "block.argentinas_delight.achuras_1click": "烤牛肠（4/5）",
```
1. 如果是同一物品的不同状态的话，需要前后文统一命名
2. 标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这些类的问题请在您翻译的所有文件中全部查明并修改。